### PR TITLE
chore: remove duplicate words

### PR DESCRIPTION
**Pull request by goofylfg**
# Description

remove duplicate words in commet

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

**Review by kittybest — APPROVED** [review]
Thanks!

File: cli/ts/utils/storage.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ export const storeContractAddress = (contractName: string, address: string, netw
 /**
  * Read a contract address from the local address store file
  * @param contractName - the name of the contract
- * @returns the contract address or a undefined it it does not exist
+ * @returns the contract address or a undefined it does not exist
```
Comment (ctrlc03) [inline cli/ts/utils/storage.ts:43]: I think this meant to read: "if it does not exist" - could you please amend?

Comment (goofylfg) [reply cli/ts/utils/storage.ts]: fixed @ctrlc03

**Comment by ctrlc03** [comment]
@goofylfg could you please amend you latest commit (and possibly squash both into one) and conform to commitlint conventions like you did on the first commit? so `chore: [..]`

**Comment by goofylfg** [comment]
> @goofylfg could you please amend you latest commit (and possibly squash both into one) and conform to commitlint conventions like you did on the first commit? so `chore: [..]`


> @goofylfg could you please amend you latest commit (and possibly squash both into one) and conform to commitlint conventions like you did on the first commit? so `chore: [..]`

@ctrlc03  done~

**Comment by ctrlc03** [comment]
> > @goofylfg could you please amend you latest commit (and possibly squash both into one) and conform to commitlint conventions like you did on the first commit? so `chore: [..]`
> 
> @ctrlc03 done~

Awesome thanks very much :)

**Review by ctrlc03 — APPROVED** [review]

**Comment by goofylfg** [comment]
@ctrlc03 Coordinator / build (pull_request) seems failed caused auto-merge not success

**Comment by ctrlc03** [comment]
> @ctrlc03 Coordinator / build (pull_request) seems failed caused auto-merge not success

there's a PR open which fixes that (basically trying to access a gh secret which PRs from fork cannot) and then we'll be able to merge yours :)

**Pull request merged**